CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ Changes since last non-beta release.
 
 *Please add entries here for your pull requests that are not yet released.*
 
+### Fixed
+- Upgrade several JS dependencies to knows security issues. [PR 1514](https://github.com/shakacode/react_on_rails/pull/1514) by [ahangarha](https://github.com/ahangarha)
```

Comment:
Upgrade several JS dependencies to fix security issues.

Reply:
My bad! Editing several times and finally pushing a wrong sentence! 😞 
Fixed ✅